src/components/Challenges/DateList/DateList.js

```diff
@@ -0,0 +1,29 @@
+import { Image, StyleSheet, Text, View } from "react-native";
+import styles from "./styles";
+
+export default function DateList({ dateList }) {
+  return (
+    <View>
+      <View style={styles.contentContainer}>
+        <Image
+          style={styles.imageStyles}
+          source={{ uri: 'https://i.ibb.co/q9t7fXV/Vector.png', width: 24, height: 26 }}
```

Comment:
If you are planing on keeping this static url then define it in constants

Reply:
This is only a temp URL, will change it in future

Reply:
Okay then lets keep it for now.